app/data-sources/rural-payments/RuralPayments.js

```diff
@@ -32,7 +32,8 @@ const customFetch = async (url, options) => {
 
   return fetch(url, {
     ...options,
-    dispatcher: proxyAgent
+    dispatcher: proxyAgent,
+    signal: AbortSignal.timeout(parseInt(process.env.RP_KITS_GATEWAY_TIMEOUT_MS))
```

Comment:
Might be worth putting a default in here to avoid errors from lack of config
```suggestion
    signal: AbortSignal.timeout(parseInt(process.env.RP_KITS_GATEWAY_TIMEOUT_MS || 5000))
```

Reply:
timeouts are relatively sensitive since our upstreams can be so slow, so think I would prefer to fail on deploy rather than run with defaults unknowingly